adapter/memQuota/memQuota_test.go

```diff
@@ -335,6 +345,54 @@ func TestReaper(t *testing.T) {
 	}
 }
 
+func TestReaperTicker(t *testing.T) {
```

Comment:
do we need this test?  I think we can assume that the Ticker works as expected.  The sleep() approach to testing it seems fragile -- and adding a callback or something just for testing seems wasteful.